src/apiflask/openapi.py

```diff
@@ -198,6 +203,14 @@ def add_response(
             content_type]['examples'] = examples
     if links is not None:
         operation['responses'][status_code]['links'] = links
+    if headers is not None:
+        operation['responses'][status_code]['headers'] = {}
+        for name, value in headers.items():
+            if isinstance(value, Field):
+                assert ma_plugin            # needed for mypy
+                assert ma_plugin.converter  # needed for mypy
+                header_obj = ma_plugin.converter.field2property(value)
```

Comment:
Can we just ignore the type check at line 212 and remove 210 and 211?

Reply:
Done it!